static/styles/zulip.css

```diff
@@ -2697,3 +2718,16 @@ img.emoji {
         margin-top: 10px;
     }
 }
+
+.high-contrast a {
+    color: hsl(200, 100%, 36%);
+}
+
+[aria-hidden="true"] {
+    visibility: hidden;
+}
```

Comment:
the [w3 website](https://www.w3.org/TR/wai-aria/states_and_properties#aria-hidden) suggests:
> It is recommended that authors key visibility of elements off this attribute, rather than change visibility and separately have to remember to update this property.

i can always make this more specific to only cover overlays?
i'll still have a look at other elements that have a `aria-hidden="true"` assignment as i have noticed a couple that should be focusable but do have it set to `true`.